src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    status: () => 'Under Construction',
+  },
```

Comment:
✅ Good job on the good naming of the slice here 👍🏽

- [ ] However, let's have a categories reducer for checking status as the code below:
```javascript
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    }
```